docs/source/command_line.rst

```diff
@@ -888,6 +888,11 @@ in developing or debugging mypy internals.
     This flag will invoke the Python debugger when mypy encounters
     a fatal error.
 
+.. option:: --plugins {MODULE|PLUGIN_FILE} ...
```

Comment:
Consider matching this with argument's [`metavar`](https://docs.python.org/3/library/argparse.html#metavar) so that the output of `mypy --help` looks the same. 

Also, `plugins` is listed before `pdb` in the config file documentation, so it may be worth using the same order here for symmetry.

Reply:
The ordering seems to be consistent already

Reply:
Hmm, maybe I'm looking at a different page. The page I'm looking at is this one: https://mypy.readthedocs.io/en/stable/config_file.html#confval-plugins, which has `plugins` before `pdb`

Reply:
you're right, isn't the ordering from [command_line.rst](docs/source/command_line.rst) preserved?